data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/export/ExportScheduler.java

```diff
@@ -32,17 +39,183 @@ public class ExportScheduler implements Runnable {
 
     private final ExecutorService executor;
 
+    private final ExportTaskManager exportTaskManager;
+    private final SnapshotManager snapshotManager;
+
     public ExportScheduler(final EnhancedSourceCoordinator sourceCoordinator,
                            final RdsClient rdsClient,
                            final PluginMetrics pluginMetrics) {
         this.pluginMetrics = pluginMetrics;
         this.sourceCoordinator = sourceCoordinator;
         this.rdsClient = rdsClient;
         this.executor = Executors.newCachedThreadPool();
+        this.exportTaskManager = new ExportTaskManager(rdsClient);
+        this.snapshotManager = new SnapshotManager(rdsClient);
     }
 
     @Override
     public void run() {
+        LOG.debug("Start running Export Scheduler");
+        while (!Thread.currentThread().isInterrupted()) {
+            try {
+                final Optional<EnhancedSourcePartition> sourcePartition = sourceCoordinator.acquireAvailablePartition(ExportPartition.PARTITION_TYPE);
+                
+                if (sourcePartition.isPresent()) {
+                    ExportPartition exportPartition = (ExportPartition) sourcePartition.get();
+                    LOG.debug("Acquired an export partition: " + exportPartition.getPartitionKey());
```

Comment:
Do not use string concatenation with loggers. Use string interpolation instead.


```suggestion
                    LOG.debug("Acquired an export partition: {}", exportPartition.getPartitionKey());
```